ONSdigital/census-services-stub
# Rm services for rh stub

# Motivation and Context
To create a stub for all RM services used by RH in order to avoid having to run the services. 

# What has Changed
This is the first iteration of the stub.

# How to test?
Run app.py in rm-services-rh-stub. Flask must be installed.  Services replaced by the stub should not be running. The environment variables for Census Respondent Home should be changed so that it knows to look for services on the the port the stubs are running on rather than the defaults. ( Details in README.md). Run Census Respondent Home and RM-Services-RH-Stub . Any IAC can be entered and it should still take you to the next screen. If want to go all the way through to eq then currently the stub is set up to take you to the LMS questionnaoire (eq_id LMS, formtype 2)

# Links
